packages/app/src/debug/DebugPageHeader.vue

```diff
@@ -38,7 +38,7 @@
         <li class="font-normal text-sm text-indigo-500 inline">
           <ExternalLink
             data-cy="debug-header-dashboard-link"
-            :href="debug.url || '#'"
+            :href="runUrl || '#'"
```

Comment:
Total nitpick - you could encapsulate the `|| '#'` case into an `if/else` conditional in the `computed` block

Reply:
Good call out.  Made the update